MapboxVisionSafety/VisionSafetyManager/VisionSafetyManager.swift

```diff
@@ -18,13 +20,27 @@ public final class VisionSafetyManager {
 
      - Returns: Instance of `VisionSafetyManager` configured with `VisionManager` instance and delegate.
      */
-    public static func create(visionManager: VisionManagerProtocol, delegate: VisionSafetyManagerDelegate? = nil) -> VisionSafetyManager {
-        let manager = VisionSafetyManager()
-        manager.native = VisionSafetyManagerNative.create(visionManager: visionManager.native, delegate: manager)
+    @available(swift, deprecated: 0.9.0, message: "This will be removed in 0.10.0. Use method create(visionManager:) instead and set delegate as property.")
```

Comment:
```suggestion
    @available(*, deprecated, message: "This will be removed in 0.10.0. Use method create(visionManager:) instead and set delegate as property.")
```